sdk/core/Azure.Core.TestFramework/src/TestProxy.cs

```diff
@@ -60,6 +60,8 @@ static TestProxy()
 
         private TestProxy(string proxyPath, bool debugMode = false)
         {
+            debugMode |= !string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable("PROXY_DEBUG_MODE"));
```

Comment:
Should we do a bool.TryParse on the string, so that users can toggle the value without deleting it entirely?

Reply:
Might also be better to move this into the getter of RecordedTestBase.UseLocalDebugProxy.